pallets/connectors/src/contract.rs

```diff
@@ -13,7 +13,7 @@
 use ethabi::{Bytes, Contract};
 use sp_std::{vec, vec::Vec};
 
-/// The ConnectorsXcmContract handle function name
+/// The ConnectorsXcmContract handle function name.
```

Comment:
`nnnit`

```suggestion
/// The ConnectorsXcmContract handle function name
```

in my previous comment I meant to say that the convention I was aware with rust one line docs is to **not** have a dot at the end. I looks kinda weird in the generated docs .

Reply:
I see, looking at the edit you said the exact opposite though. I have no strong opinion and will revert the changes then.

Reply:
> I see, looking at the edit you said the exact opposite though. I have no strong opinion and will revert the changes then.


Not sure if I misunderstand you but I edited the comment to correct it so that it say the right thing, since I noticed I had missed the **not** which said the opposite of what I meant. Mea culpa!

Reply:
I was phrasing poorly - sorry. totally got what you were saying. Looking at Substrate Doc comments, there are actually periods for one liners. I also think it is much more consistent than leaving the existence of periods up to the number of lines. Can we just leave it as is?

Reply:
Sure! :) This is absolutely not important, sorry for all the noise